app/javascript/pages/home.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import '@patternfly/react-core/dist/styles/base.css';
+import { Gallery, GalleryItem, Text } from '@patternfly/react-core';
+import ArticleCard from '../components/articleCard.js';
+import previewImage from '../../assets/images/img1.png';
+import colors from '../utils/colors.js';
+import { useNavigate } from 'react-router-dom';
+import articles from '../utils/articles.js';
+const styles = {
+  headerText: {
+    fontWeight: 'bolder',
+    textAlign: 'center',
+    flex: 1,
+    color: colors.appDarkBlue,
+    fontSize: '50px'
+  }
+};
+const Home = () => {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <Text style={styles.headerText}>Articles for you!!!</Text>
```

Comment:
Can we drop this name to more specific, like 'Newsletters' or just 'Articles'? What do you think @Ron-Lavi @girijaasoni?